keyboards/ergodox_ez/keymaps/dvorak_42_key/keymap.c

```diff
@@ -493,19 +502,19 @@ const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
             if (record->event.pressed) {
```

Comment:
(because I can't tag it elsewhere)

Just a heads up, we've depreciated the macro function a while ago. While it still works, it may be a good idea to move this stuff to `process_record_user` in the future.  You don't need to do this now, but I wanted to mention it, so that you can think about it, if you want to do so.

Reply:
thank you, i'll factor it in to my next change